include/text.php

```diff
@@ -692,51 +694,58 @@ function attribute_contains($attr,$s) {
  *
  * @global App $a
  * @global dba $db
+ * @global array $LOGGER_LEVELS
  * @param string $msg
  * @param int $level
  */
-function logger($msg,$level = 0) {
-	// turn off logger in install mode
+function logger($msg, $level = 0) {
 	global $a;
 	global $db;
 	global $LOGGER_LEVELS;
 
-	if(($a->module == 'install') || (! ($db && $db->connected))) return;
-
-	if (count($LOGGER_LEVELS)==0){
-		foreach (get_defined_constants() as $k=>$v){
-			if (substr($k,0,7)=="LOGGER_")
-				$LOGGER_LEVELS[$v] = substr($k,7,7);
-		}
-	}
-
 	$debugging = get_config('system','debugging');
-	$loglevel  = intval(get_config('system','loglevel'));
 	$logfile   = get_config('system','logfile');
+	$loglevel = intval(get_config('system','loglevel'));
 
-	if((! $debugging) || (! $logfile) || ($level > $loglevel))
+	// turn off logger in install mode
+	if (
+		$a->module == 'install'
+		|| ! ($db && $db->connected)
+		|| ! $debugging
+		|| ! $logfile
+		|| $level > $loglevel
+	) {
 		return;
+	}
+
+	if (count($LOGGER_LEVELS) == 0) {
+		foreach (get_defined_constants() as $k => $v) {
+			if (substr($k, 0, 7) == "LOGGER_") {
+				$LOGGER_LEVELS[$v] = substr($k, 7, 7);
+			}
+		}
+	}
 
 	$process_id = session_id();
 
-	if ($process_id == "")
+	if ($process_id == '') {
 		$process_id = get_app()->process_id;
+	}
 
 	$callers = debug_backtrace();
-	$logline =  sprintf("%s@%s\t[%s]:%s:%s:%s\t%s\n",
-				 datetime_convert(),
-				 $process_id,
-				 $LOGGER_LEVELS[$level],
-				 basename($callers[0]['file']),
-				 $callers[0]['line'],
-				 $callers[1]['function'],
-				 $msg
-				);
+	$logline = sprintf("%s@%s\t[%s]:%s:%s:%s\t%s\n",
+		datetime_convert(),
+		$process_id,
+		$LOGGER_LEVELS[$level],
+		basename($callers[0]['file']),
+		$callers[0]['line'],
+		$callers[1]['function'],
+		$msg
```

Comment:
This indention confuses me. Since the lines belong to the "sprintf" function I would always indent them one tab more.